client/src/stores/historyItemsStore.ts

```diff
@@ -0,0 +1,95 @@
+/**
+ * Requests history items by reacting to changes of filter props passed
+ * to the history items provider used in the history panel.
+ */
```

Comment:
I feel like this comment is either outdated, or unclear: I fail to find the mentioned provider and it's connection to this store